src/tf/BaseTFClient.ts

```diff
@@ -12,7 +12,6 @@ export default class BaseTFClient extends EventEmitter {
     { transform?: Transform; cbs: ((tf: Transform) => void)[] }
   > = {};
   republisherUpdateRequested = false;
-  _isDisposed = false;
```

Comment:
This was erroneously in the shared base class when only used by one of the children.